config/pipeline.yaml

```
@@ -2308,6 +2325,7 @@ scheduler:
     rules:
       tree:
         - next:master
+        - sashal-next:linus-next
```

```suggestion
        - sashal-next
```

Minor nit, we only build the `linux-next` branch for this tree. Keeping it as-is makes sense if we plan to build other branches in the future, though.

---

Good catch, thanks! I'll squash it before merging this PR